content/blog/noetic-eol/index.md

```diff
@@ -0,0 +1,166 @@
+---
+title: "Upcoming ROS 1 End of Life"
+author:  Kat Scott 
+subtitle: Migration Paths for ROS 1 Users
+created_at: 2024-12-16 16:21:05 -0700
+kind: article
+header_class:  bg-photo 
+header_image:  /imgs/eol-bg.png 
+---
+
+## ROS 1 Noetic Ninjemys goes End-of-Life 2024-05-31
+
+When we [announced](https://www.openrobotics.org/blog/2020/5/23/noetic-ninjemys-the-last-official-ros-1-release) ROS 1 Noetic Ninjemys on May 23rd, 2020 we made it very clear that it would be the last official ROS 1 distribution. 
```

Comment:
When I read this I get "per my previous message" vibes. I would recommend reducing it to:

```suggestion
ROS Noetic Ninjemys, the [last ROS 1 release](https://www.openrobotics.org/blog/2020/5/23/noetic-ninjemys-the-last-official-ros-1-release), will reach End of Life in May 2025.
```